tests/sentry/backup/test_correctness.py

```diff
@@ -165,7 +267,6 @@ def test_bad_failing_comparator_field(tmp_path):
     assert findings[0].kind == "DateUpdatedComparator"
 
 
-@django_db_all(transaction=True, reset_sequences=True)
```

Comment:
why were all of these removed?

Reply:
They were erroneous. These tests don't actually require the database to be reset, since we are just comparing raw JSON without touching the DB. Resetting the DB is a super slow operation (~5 seconds per test, and prevents test parallelization to boot), so this is a nice development speed win.